src/components/Icon.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
```

Comment:
Not sure if this component is needed since we just import the icons from react feather

Reply:
Banner needs it, will remove on refactor